libs/platform/src/lib/table/table.component.html

```diff
@@ -260,13 +262,13 @@
                                             | async
                                             | selectionCellStyles : _rtl : _semanticHighlightingColumnWidth
                                     "
+                                    (click)="_toggleMultiSelectRow(row, $event)"
+                                    (keydown.enter)="_toggleMultiSelectRow(row, $event)"
+                                    (keydown.space)="_toggleMultiSelectRow(row, $event)"
```

Comment:
We have a directive called `fdkClick` which does exactly this, maybe it is worth using that instead?